content/docs/ref/pr.md

```diff
@@ -6,7 +6,7 @@ cml pr create [options] <pathspec>...
 
 Commit specified files to a new branch and create a pull request. If sending a
 report afterwards, consider using
-[`cml comment update --target=pr`](/doc/ref/comment#update).
+[`cml comment update`](/doc/ref/comment#update).
```

Comment:
just double-checking: the `--target=pr` flag is auto-inferred here, right?

Reply:
Yes, by default the target is automatically decided based on [dark magic of all sorts](https://github.com/iterative/cml/blob/506e4e8d60e48e2ee4e99cfe9ff38ad6ec6a8f7f/src/commenttarget.js#L29-L54); see https://github.com/iterative/cml/pull/1228.

Reply:
Although it's not clear to me whether the documentation is suggesting users to comment on the ancillary pull request or the main one. 🤔